src/Infrastructure/Services/UserService.cs

```diff
@@ -183,6 +187,28 @@ public async Task<Response<NhsLoginResponse>> NhsLoginAsync(string code, string
             }
         }
 
+        private async Task SendContentfulEmailAsync(string emailName, string emailRecipient, string selectedLanguage,
+            string firstName = null)
+        {
+            var contentfulEmail = await _contentfulService.GetContentfulEmailAsync(emailName, selectedLanguage);
+            string htmlContent = _richTextToHtmlConverter.Convert(contentfulEmail.EmailBody);
+
+            if (!string.IsNullOrEmpty(firstName))
+            {
+                var template = Handlebars.Compile(htmlContent);
+                var data = new
+                {
+                    firstName =
+                        System.Threading.Thread.CurrentThread.CurrentCulture.TextInfo.ToTitleCase(firstName.ToLower()),
```

Comment:
Use the selected locale to do text manipulation `selectedLocale.TextInfo.ToTitleCase(firstName)`